examples/Umbraco.AuthorizedServices.TestSite/appsettings.json

```diff
@@ -25,9 +25,8 @@
     },
     "AuthorizedServices": {
       "TokenEncryptionKey": "",
-      "Services": [
-        {
-          "Alias": "assetBank",
+      "Services": {
+        "assetBank": {
```

Comment:
Note that this changed from an array to an object and failing to change this will result in the array indexes being used as alias.